.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: NEURON Bug Report
+about: Detailed report to describe NEURON related bugs
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+## Context
+
+### Overview of the issue
+
+[Provide more details regarding the issue]
+
+### Expected result/behavior
+
+[Describe what is the expected result and or behavior]
+
+### NEURON setup
+ 
+ - Version: [e.g. master branch / 7.8.2]
+ - Installation method [e.g. cmake / pip / Windows Installer / macOS installer]
+ - OS + Version: [e.g. Ubuntu 20.04]
+ - Compiler + Version: [e.g. gcc-9]
+ 
+## Minimum working example
+
+MWE that can be used for reproducing and testing, e.g: 
+```python
+from neuron import h
+
+def test_soma():
+    h('''create soma''')
+    assert h.soma is not None
+
+    h.soma.L = 5.6419
+    assert h.soma.L == 5.6419
+
```

Comment:
Personally, I'd rather not have anything that gives examples of using HOC where Python would work just fine.

What about:

```python
from neuron import h

def test_area():
    soma = h.Section(name="soma")
    soma.L = soma.diam = 10
    assert 314.159 < sum(seg.area() for seg in soma) < 314.16

test_area()

Reply:
Sure, great idea.